_inc/lib/plans.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Plans Library
+ *
+ * Fetch plans data from WordPress.com
+ */
+
+use Automattic\Jetpack\Connection\Client;
+
+class Jetpack_Plans {
```

Comment:
Why not use [`class.jetpack-plan.php`](https://github.com/Automattic/jetpack/blob/7f96a8cf205f04a82624d046aa2e2a6684045816/class.jetpack-plan.php)?

Reply:
That file does something completely different, IIUC -- it stores (and syncs) information about the _current_ plan, whereas this little library is supposed to provide information about any plan given its slug.

Reply:
Hmm right, makes sense. Do you think there could be another class name that would be more clear? `Jetpack_Plan` vs `Jetpack_Plans` are basically identical.

Reply:
`Jetpack_Plan` really seems the most descriptive here :sweat_smile: `Jetpack_Plan_Information` mabe? `Jetpack_Plan_Data`? TBH not a big fan of either, since `_Infomation` and `_Data` sounds a bit unnecessary/strawman-ish in this context.

Reply:
> Yeah, it's a bit confusing, but maybe we can keep the class names and just add some comments to the main class comment in both classes to explain what you described above? This way no one will have to hunt for that information.

Makes sense, I'll do that :+1: 

> Another approach would have been to add your methods to the existing class. That may have been less confusing overall? I could imagine a `Jetpack_Plan` class being used to gather info about a plan in general, whether that's the plan you are currently using on your site, or _could be using_ at some point.

TBH, looking at what both classes do, it seems like that would be a total mix of concerns, so I'd keep them separate. While static, the `Jetpack_Plan` class really seems to represent the plan that the site is currently on, with methods such as `refresh_from_wpcom()` to sync that information from WP.com. The `Jetpack_Plans` helper I'm introducing OTOH is essentially just an interface to the plans information endpoint; it also differs from the `Jetpack_Plan` endpoint in that its methods are side-effect free.

Reply:
Added comments to the relevant classes in 994030e80.